srtcore/group_backup.h

```diff
@@ -79,6 +79,7 @@ namespace groups
             : m_stateCounter() // default init with zeros
             , m_activeMaxWeight()
             , m_standbyMaxWeight()
+            , m_rateEstimate(AF_INET6) // XXX Probably the whole solution is wrong
```

Comment:
Why INET6 here?

Reply:
The actual problem here is that potentially a group may contain both IPv4 and IPv6 links. This means that the number of actual sent bytes (including all headers) isn't constant. As a "worst case", the calculation for the group uses the bigger of two values, so in case of IPv4 the stats could be a bit overestimated.

I have no idea how to solve it, actually. This value here is a stub. I can change it to AF_INET, which could restore the old behavior, although won't solve the problem.

Reply:
Right, makes total sense. Can you place your clarification in the comment to that initialization to not forget in the future?